src/CosmosDB/deploy.ps1

```diff
@@ -0,0 +1,56 @@
+﻿$extensionPath=$args[0]
```

Comment:
can we generalize this and move into a separate folder - e.g. tools?

Reply:
ack